vmdb/app/models/configured_system_foreman.rb

```diff
@@ -6,6 +6,11 @@ class ConfiguredSystemForeman < ConfiguredSystem
   belongs_to :customization_script_medium
   belongs_to :customization_script_ptable
 
+  delegate :name, :to => :configuration_location,     :prefix => true
+  delegate :name, :to => :configuration_organization, :prefix => true
+  delegate :name, :to => :operating_system_flavor,    :prefix => true
+  delegate :name, :to => :provider,                   :prefix => true
```

Comment:
These last 2 delegates belong back on the configured_system model where those relationships are defined and I think you need `:allow_nil => true` on these as well.